src/modules/api/user/link/link.controller.ts

```diff
@@ -1,7 +1,12 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Body, Param } from '@nestjs/common';
 import { LinkService } from './link.service';
 
 @Controller('users/:userId/links')
 export class LinkController {
-  constructor(private linkService: LinkService) {}
+  constructor(private linkService: LinkService) { };
+  @Post()
+  async create(@Body() body, @Param() params) {
+     const result = await this.linkService.create(body,params.userId);
+     return { "status":"OK" , "result":result};
```

Comment:
return type should be strictly typed.
Please create interface structure for this.